fe/fe-core/src/main/java/com/starrocks/lake/LakeTableHelper.java

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.lake;
+
+import com.google.common.base.Preconditions;
+import com.starrocks.alter.AlterJobV2Builder;
+import com.starrocks.alter.LakeTableAlterJobV2Builder;
+import com.starrocks.catalog.OlapTable;
+import com.starrocks.server.GlobalStateMgr;
+
+public class LakeTableHelper {
+    static boolean deleteTable(long dbId, OlapTable table, boolean replay) {
+        Preconditions.checkState(table.isCloudNativeTableOrMaterializedView());
+        table.removeTableBinds(replay);
+        // LakeTable's data cleaning operation is a time-consuming operation that may need to be repeated
+        // many times before it succeeds, so we chose to let the recycle bin perform this work for two purposes:
+        // 1. to avoid blocking the user's session for too long
+        // 2. to utilize the recycle bin's delete retry capability to perform retries.
+        GlobalStateMgr.getCurrentState().getRecycleBin().recycleTable(dbId, table, false);
+        return true;
+    }
+
+    static boolean deleteTableFromRecycleBin(long dbId, OlapTable table, boolean replay) {
+        Preconditions.checkState(table.isCloudNativeTableOrMaterializedView());
+        table.removeTableBinds(replay);
```

Comment:
This has been called from deleteTable above, seems useless ?

Reply:
`deleteTable()` will only be called for forced dropped tables.

Reply:
If a table is not force dropped, then `table.delete()` is not called, so we still need to `revoveTableBindings()` here. Of course, if a table is force drop dropped, then both `table.delete()` and `table.deleteFromRecycleBin()` are called, in which case `removeTableBinds()` does get called repeatedly.